src/server/enrollment-system.js

```diff
@@ -341,9 +341,6 @@ function childToDependentInfo(child) {
  */
 function childToDependentFinancialsInfo(child) {
   const incomes = resourceToIncomeCollection(child);
-  if (!incomes) {
```

Comment:
Yep, sounds good. I would just ask them to test cases where:
1. No veteran income info is provided and spouse/dependent identity is, but no spouse/dependent income info
2. Veteran income info is provided and spouse/dependent identity is, but no spouse/dependent income info